modules/shared/components/molecules/imageCaption/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import classNames from 'classnames';
+import type { FC, ReactElement } from 'react';
+import type { IImgCaption } from './IImgCaption';
+import styles from './imageCaption.module.css';
+
+const ImageCaption: FC<IImgCaption.IProps> = ({
+  imgCaptionLetter,
+  imgCaption,
+}): ReactElement => {
+  // make letter and devider (|) hidden when not pass letter as a props
+  const classes = classNames({
+    ' hidden ': !imgCaptionLetter,
+  });
+  return (
+    <div className={styles.captionContainer}>
+      <div className={styles.captionContent}>
+        <span data-testid="captionLetter" className={classes}>
+          {imgCaptionLetter}
+        </span>
+
+        <div
+          data-testid="captionDivider"
```

Comment:
I noticed that you aren't using that test id in your test file so what's the usage of it?

Reply:
I used it before I switched tests to snapshots, I will remove it .